service/frontend/workflowHandler.go

```diff
@@ -115,6 +115,7 @@ var (
 	errQueryDisallowedForDomain                   = &gen.BadRequestError{Message: "Domain is not allowed to query, please contact cadence team to re-enable queries."}
 	errClusterNameNotSet                          = &gen.BadRequestError{Message: "Cluster name is not set."}
 	errEmptyReplicationInfo                       = &gen.BadRequestError{Message: "Replication task info is not set."}
+	errEmptyQueueType                             = &gen.BadRequestError{Message: "Queue type is not set."}
```

Comment:
Is this error only used in adminHandler?

Reply:
Yes